content/logs/processing/attributes_naming_convention.md

```diff
@@ -0,0 +1,188 @@
+| `http.useragent_details.device.family`  | `string` | The Device family reported by the user-agent.  |
+
+## Source code
+
+Related to the data used when a log or an error is generated via a logger in a custom application. All attributes are prefixed either by `logger` or `error`.
+ 
+| **Fullname**         | **Type** | **Description**                                                  |
+| :---                 | :---     | :----                                                            |
+| `logger.name`        | `string` | The name of the logger.                                          |
+| `logger.thread_name` | `string` | The name of the current thread when the log is fired.            |
+| `logger.method_name` | `string` | The class method name.                                           |
+| `error.kind`         | `string` | The error type or kind (or code is some cases).                  |
+| `error.message`      | `string` | A concise, human-readable, one-line message explaining the event |
+| `error.stack`        | `string` | The stack trace or the complementary information about the error |
+
+Typical integrations relying on these attributes are: *Java*, *NodeJs*, *.NET*, *Golang*, *Python*, etc.
+
+## Database
+
+Database related attributes are prefixed by `db`.
+
+| **Fullname**   | **Type** | **Description**                                                                                                                       |
+| :---           | :---     | :----                                                                                                                                 |
+| `db.instance`  | `string` | Database instance name. E.g., In java, if the `jdbc.url="jdbc:mysql://127.0.0.1:3306/customers"`, the instance name is `customers`.   |
+| `db.statement` | `string` | A database statement for the given database type. E.g., for mySQL: `"SELECT * FROM wuser_table";` for Redis: `"SET mykey 'WuValue'"`. |
+| `db.operation` | `string` | The operation that was performed (“query”, “update”, “delete”,...).                                                                   |
+| `db.user`      | `string` | User that performs the operation.                                                                                                     |
+
+Typical integrations relying on these attributes are: [Cassandra][15], [MySQL][16], [RDS][17], [Elasticsearch][18], etc...
+
+## Performance
+
+Performance metrics.
+
+| **Fullname** | **Type** | **Description**                                                                                   |
+| :---         | :---     | :----                                                                                             |
+| `duration`   | `number` | A duration of any kind in **nanoseconds**: HTTP response time, database query time, latency, etc. |
+
+
+We advise you to rely or at least remap on this attribute as Datadog displays and uses it in the best way possible.
```

Comment:
Be more specific about "best way possible" — this is a bit confusing, what does it mean?